File: Data Structures & Algorithms/duplicate-integer/submission-12.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        setNums = set()
        for num in nums:
            if num in setNums:
                return True
            setNums.add(num)
        return False

# class Solution:
#     def hasDuplicate(self, nums: List[int]) -> bool:
#         return len(set(nums)) < len(nums)

File: Data Structures & Algorithms/remove-element/submission-12.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int n = nums.length, l = 0, r =0;
        for (r = 0; r < n; r++) {
            if (nums[r] != val) {
                nums[l] = nums[r];
                l++;
            }
        }
        return l;
    }
}